robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -527,11 +554,9 @@ function sendPdsCommand (cmd) {
   pds_command_publisher.publish(command)
```

Comment:
Small thing that you could do is change it here for "Sending cmd to PDS". because it doesnt use a teensy. Or you can say atmega328p

Reply:
Fixed